src/components/avatar/index.js

```diff
@@ -314,17 +314,17 @@ export default class Avatar extends PureBaseComponent {
         style={[this.getContainerStyle(), containerStyle]}
         testID={testID}
         onPress={onPress}
-        accessible
+        accessible={!_.isUndefined(onPress)}
         accessibilityLabel={'Avatar'}
         accessibilityRole={onPress ? 'button' : 'image'}
         {...this.extractAccessibilityProps()}
       >
         <View
           style={[this.getInitialsContainer(), {backgroundColor}, hasImage && this.styles.initialsContainerWithInset]}
         >
-          <Text numberOfLines={1} style={[{fontSize}, this.styles.initials, {color}]}>
+          {label && <Text numberOfLines={1} style={[{fontSize}, this.styles.initials, {color}]}>
```

Comment:
this is what we talked about last week.
I believe that if you'll pass `''` (empty string) to label it will throw an error, can u please verify

Reply:
You are right 🤦‍♂ 
I'll fix this tomorrow since I cannot test this on iOS ATM.

Reply:
You can test on a simulator, the fix is simple, as discussed we need to use _.isUndefined

Reply:
Fixed